rust/ql/test/query-tests/security/CWE-079/axum/main.rs

```diff
@@ -0,0 +1,20 @@
+use axum::{extract::Query, response::Html, routing::get, Router};
+
+#[derive(serde::Deserialize)]
+struct GreetingParams {
+    name: String,
+}
+
+async fn greet_handler(Query(params): Query<GreetingParams>) -> Html<String> {
+    let html_content = format!("<p>Hello, {}!</p>", params.name);
+    Html(html_content) // $ MISSING: Alert[rust/xss]
```

Comment:
Is the plan to add sink models for this as part of the `axum` modelling issue?

Also we model [poem](https://docs.rs/poem/latest/poem/) for some similar queries, should we model (and optionally test) that here as well?